.circleci/config.yml

```diff
@@ -159,7 +159,7 @@ jobs:
             - prod-build-{{ .Branch }}-{{ .Revision }}
       - run:
           command: | # if the build job starts failing with exit code 137 (running out of memory), add the build_all.sh option -c "half" to cut the concurrency
-            [ -f "client/$BUILD_DIR/InfoBase/app/app-a11y-en.min.js" ] || (cd client && ./deploy_scripts/build_all.sh -m 256 -c "half")
+            [ -f "client/$BUILD_DIR/InfoBase/app/app-a11y-en.min.js" ] || (cd client && ./deploy_scripts/build_all.sh -m 256 -c "none")
```

Comment:
Was it still running out of memory even with the final, fixed, version of `build_all.sh` and `half`?

Reply:
Yup half runs out of memory often